Dockerfile

```diff
@@ -1,14 +1,13 @@
-FROM mikefarah/yq
+FROM mikefarah/yq as builder
```

Comment:
We could add copyright headers there (they were not there before your changes so..)

Reply:
ok